python/nano/src/bigdl/nano/tf/keras/Model.py

```diff
@@ -21,3 +21,31 @@
 class Model(tf.keras.Model):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+
+    def quantize(self, model, calib_dataset, batch=1, val_dataset=None,
+                 metric: int = 0, backend='inc', conf=None, approach='ptsq', strategy='bayesian',
+                 accuracy_criterion=None, timeout=0, max_trials=1, **kwargs):
+        if backend == 'inc':
+            from bigdl.nano.quantization import QuantizationINC
+            from neural_compressor.experimental import common
+            quantizer = QuantizationINC(framework='tensorflow', conf=conf, approach=approach,
+                                        strategy=strategy, accuracy_criterion=accuracy_criterion,
+                                        timeout=timeout, max_trials=max_trials, inputs=['input_1'],
+                                        outputs=['predictions'])
```

Comment:
What are the `inputs` and `outputs` values?

Reply:
`inputs, outputs` are input/output names for tensorflow. Values here are only for debugging, I will change them later.